ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -482,6 +482,17 @@ def Relu6(features, name="Relu6"):
 )
 
 
+Softmax = to_ivy_arrays_and_back(
+    map_raw_ops_alias(
+        tf_frontend.keras.activations.softmax,
+        kwargs_to_update={
+            "logits": "x",
+            "": "axis",
```

Comment:
I'm not really sure if that is right, I wanted to invalidate the `axis` argument as it not needed as described in the [docs](https://www.tensorflow.org/api_docs/python/tf/raw_ops/Softmax)